AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Sandbox.py

```diff
@@ -159,3 +160,17 @@ def test_MaterialEditorBasicTests(
             enable_prefab_system=False,
         )
 
+
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+@pytest.mark.parametrize("launcher_platform", ['windows_editor'])
+class TestAutomation(EditorTestSuite):
+
+    enable_prefab_system = False
+
+    @pytest.mark.test_case_id("C36529666")
```

Comment:
Case ID is no longer hooked up to a reporting system, but you are welcome to continue using it for your own tracking.